src/librbd/AioObjectRequest.h

```diff
@@ -331,11 +331,14 @@ class AioObjectRemove : public AbstractAioObjectWrite {
 
 class AioObjectTrim : public AbstractAioObjectWrite {
 public:
+  // we'd need to only conditionally specify if a post object map
```

Comment:
@smithfarm 
I believe multi-line comment should be inside /* */
Please replace we'd with we would